src/modules/45-dashboards/components/AidaChatInput/__tests__/AidaChatInput.test.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { act, fireEvent, render, RenderResult, screen } from '@testing-library/react'
+import { TestWrapper } from '@common/utils/testUtils'
+import AidaChatInput from '..'
+
+const renderComponent = (onEnter: (value: string) => void = jest.fn()): RenderResult =>
+  render(
+    <TestWrapper>
+      <AidaChatInput onEnter={onEnter} />
+    </TestWrapper>
+  )
+
+describe('AidaChatInput', () => {
+  test('it should display placeholder text in input', async () => {
+    const { container } = renderComponent()
+    expect(screen.getByPlaceholderText('common.csBot.askAIDA')).toBeInTheDocument()
+    expect(container.querySelector('svg')?.getAttribute('data-icon')).toEqual('arrow-right')
```

Comment:
Not a bug, but just FYI that `toHaveAttribute` is perfect for these things.
```suggestion
    expect(container.querySelector('svg')).toHaveAttribute('data-icon', 'arrow-right')
```